lib/roles/admin/admin_screen.dart

```diff
@@ -10,24 +14,69 @@ class AdminScreen extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     Credentials? credentials = ref.watch(authProvider).credentials;
-    AuthState? authState = ref.watch(authProvider);
+//    AuthState? authState = ref.watch(authProvider);
     return Scaffold(
       appBar: AppBar(title: const Text('Admin - Details Screen auth0')),
       body: Column(
         children: [
           Center(
             child: ElevatedButton(
-                onPressed: () =>
-                    print(authState!.credentials!.scopes.toString()),
-                child: credentials != null
-                    ? Column(
-                        children: [
-                          Text(credentials.accessToken),
-                          Text(credentials.scopes.toString()),
-                        ],
-                      )
-                    : const Text('not logged')),
+                onPressed: () async {
+                  try {
+                    final Dio dio = Dio(
+                      BaseOptions(
+
+                          ///vvvv router IP vvvv///
```

Comment:
I think these comments are unnecessary.

Reply:
Okay. I put that big comment because I considered it important for you to test the project locally since not entering the local IP address generates a connection error. It was just an informative reminder. This comment will be removed.